lib/web/mage/dropdown_old.js

```diff
@@ -26,6 +27,21 @@ define([
             if (event.keyCode == ESC_KEY_CODE) { //eslint-disable-line eqeqeq
                 $('[data-toggle=dropdown].active').trigger('close.dropdown');
             }
+            if (event.keyCode == ENTER_KEY_CODE) { //eslint-disable-line eqeqeq
+                event.preventDefault();
+                $('.action-accept').click();
+            }
+
+
+
+        })
+        .ready(function() {
+          $(window).keydown(function(event){
```

Comment:
What this code should do? Looks like you try to prevent action that shouldn't be at all. Usually, when I open some page and press the enter button - nothing happens, because `window` doesn't have any default behavior. If in this case, this is not true maybe we should fix it in another place